h/static/scripts/annotation-ui-controller.js

```diff
@@ -21,7 +21,7 @@ function AnnotationUIController($rootScope, $scope, annotationUI) {
   });
 
   $rootScope.$on(events.ANNOTATION_DELETED, function (event, annotation) {
-    annotationUI.removeSelectedAnnotation(annotation);
+    annotationUI.removeSelectedAnnotation(annotation.id);
```

Comment:
This pr changes `removeSelectedAnnotation()` to take an `id` not the whole annotation, so this is just updating the call.